packages/components/src/Page/Page.test.tsx

```diff
@@ -106,6 +108,23 @@ describe("When actions are provided", () => {
     fireEvent.click(getByText("Second Do"));
     expect(handleSecondaryAction).toHaveBeenCalledTimes(1);
   });
+
+  describe("getActionProps", () => {
+    it("given action props, it returns the correct props", () => {
+      const buttonActionProps = {
+        label: "Label",
+      } as ButtonProps;
+      const buttonActionWithRefProps = {
+        label: "Label",
+        ref: { current: null },
+      } as ButtonActionProps;
```

Comment:
for this and the one above it, do we need to cast these objects as these types or can we let Typescript figure it out for us?

Reply:
It's definitely not needed. TS can figure it out no problem. It was more just readability for the person looking at the test to see that it's explicitly being cast as this type.

But I'm happy to remove it.

Reply:
gotcha. personally I prefer to avoid casting unless Typescript is in a situation where it can't figure it out, so if it's all the same to you - let's remove it